assignment7.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Assignment7</title>
+</head>
+<style>
+    li{
+        margin: 7px;
+    }
+</style>
+<body>
+    <h1>My Shopping List</h1>
+    <form>
+    <label>Enter a new item:</label>
+    <input id="item">
+    <button type="button" onclick="additem()">Add item</button>
+    </form>
+    <ul id="ul"></ul>
+    <script>
+        function additem()
+        {
+            let name=document.getElementById("item").value + " ";
```

Comment:
better to add style instead of adding space after text

Reply:
ok.modified